static/app/views/organizationGroupDetails/eventToolbar.tsx

```diff
@@ -97,6 +98,7 @@ class GroupEventToolbar extends Component<Props> {
   }
 
   render() {
+    const is24Hours = use24Hours();
```

Comment:
unrelated to this PR but we should change the name of this sound it doesn't sound like a react hook